0.wave1_data/1.cellprofiler_ic_processing/cellprofiler_environment.yml

```diff
@@ -0,0 +1,26 @@
+name: cp4
+
+channels:
+- conda-forge
+- anaconda
+- bioconda
+- defaults
+
```

Comment:
```suggestion
```

Reply:
I'm not used to seeing these extra spaces!

Reply:
also, aren't these files usually indented by two spaces?

```yaml
# Like this
dependencies:
  - python=3.8

# Not like this
dependencies:
- python=3.8
```

This is a minor style comment.

Reply:
I just copied and pasted what was on the CellProfiler wiki for conda install. This is an easy style change I will implement.